discovery-provider/src/queries/queries.py

```diff
@@ -1982,3 +1983,130 @@ def get_top_genre_users():
 
 
         return api_helpers.success_response({ 'user_ids': user_ids })
+
+
+
+# Get the tracks that are 'children' remixes of the requested track
+# The results are sorted by if the original artist has reposted or saved the track
+@bp.route("/remixes/<int:track_id>/children", methods=("GET",))
+def get_remix_tracks(track_id):
+
+    db = get_db_read_replica()
+    with db.scoped_session() as session:
+        parent_track = session.query(Track).filter(
+            Track.is_current == True,
+            Track.track_id == track_id
+        ).first()
+        
+        if (parent_track == None):
+            return api_helpers.error_response("Invalid track_id provided", 400)
+
+        track_owner_id = parent_track.owner_id
+        base_query = session.query(Track).join(
```

Comment:
super nit, i like it when we do this formatting
```
x = (
  session.query(x)
  .join(y)
  .outerjoin(z)
)
```

Reply:
👍 updated!